jobmaster/dm/task_manager_test.go

```diff
@@ -236,25 +229,24 @@ func TestCheckAndOperateTasks(t *testing.T) {
 	}
 	taskManager.UpdateTaskStatus(dumpStatus1)
 	taskManager.UpdateTaskStatus(dumpStatus2)
-	require.NoError(t, taskManager.checkAndOperateTasks(context.Background(), job))
+	require.NoError(t.T(), taskManager.checkAndOperateTasks(context.Background(), job))
```

Comment:
we can check mockAgent if it's received some call, to align the worker status

Reply:
It will no call mockAgent here. If it calls unexpected, mockAgent will panic in the test.